src/containers/ManageListingsPage/ManageListingsPage.js

```diff
@@ -173,16 +198,25 @@ export const ManageListingsPageComponent = props => {
                 key={l.id.uuid}
                 listing={l}
                 isMenuOpen={!!listingMenuOpen && listingMenuOpen.id.uuid === l.id.uuid}
-                actionsInProgressListingId={openingListing || closingListing}
+                actionsInProgressListingId={openingListing || closingListing || discardingDraft}
                 onToggleMenu={onToggleMenu}
                 onCloseListing={onCloseListing}
                 onOpenListing={handleOpenListing}
+                onDiscardDraft={openDiscardDraftModal}
                 hasOpeningError={openingErrorListingId.uuid === l.id.uuid}
                 hasClosingError={closingErrorListingId.uuid === l.id.uuid}
+                hasDiscardingError={discardingErrorListingId.uuid === l.id.uuid}
```

Comment:
No need to modify things, but this pattern of has<X>Error starts to be somewhat stretched at this point.